tests/filesystem_tests/test_hdfs_handler.py

```diff
@@ -24,8 +24,6 @@ def test_read_bytes(self):
                 tmpfile.write(self.test_string_b)
             with handler.open(self.tmpfile_name, "rb") as f:
                 self.assertEqual(self.test_string_b, f.read())
-            with handler.open(self.tmpfile_name, "rb") as f:
-                self.assertEqual(self.test_string_b, f.readline())
```

Comment:
It would be nice if we keep this test, as it can be a test that checks whether the file object is properly wrapped by TextIOWrapper.

Reply:
I think what you meant is tested at https://github.com/chainer/chainerio/blob/09f3194a5419530221ead29da342d023ae03ca1a/tests/filesystem_tests/test_hdfs_handler.py#L35
This test file is opened as "rb".